spec-dtslint/operators/bufferToggle-spec.ts

```diff
@@ -0,0 +1,32 @@
+import { of, NEVER } from 'rxjs';
+import { bufferToggle } from 'rxjs/operators';
+
+it('should infer correctly', () => {
+  const o = of(1, 2, 3).pipe(bufferToggle(of('a', 'b', 'c'), value => of(new Date()))); // $ExpectType Observable<number[]>
+});
+
+it('should support Promises', () => {
+  const promise = new Promise<string>(() => 'a');
```

Comment:
I would write this as `Promise.resolve('a')`.

Reply:
Done.